client/state/marketplace/types.ts

```diff
@@ -18,6 +18,12 @@ export interface IPurchaseFlowState {
 	reasonForPluginInstallationStatus: string | null;
 	isPluginInstalledAlongWithTransfer: boolean | null;
 }
+
+export interface IBillingIntervalState {
+	interval: string;
```

Comment:
We can also replace `interval: string` with `interval: IntervalLength.MONTHLY | IntervalLength.ANNUALLY`?

Reply:
Great suggestion. I will fix this too.